WalletWasabi.Gui/ViewModels/MainWindowViewModel.cs

```diff
@@ -111,6 +110,34 @@ public void Initialize()
 			}
 		}
 
+		private void InitializeLockScreen (UiConfig uiConfig)
+		{
+			uiConfig
+				.WhenAnyValue(x => x.LockScreenActive)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(x =>
+				{
+					if (x)
+					{
+						CheckLockScreenType(uiConfig.LockScreenPinHash);
```

Comment:
I don't know if it's a good idea to access local variable inside an rx subscribe. @molnard ?

Reply:
@molnard any advice on here?